src/client/src/ui/blotter/components/Blotter.tsx

```diff
@@ -8,6 +8,24 @@ import { Trade, TradeStatus } from 'rt-types'
 import BlotterGrid from './BlotterGrid'
 import BlotterHeader from './BlotterHeader'
 import { columnDefinitions, DEFAULT_COLUMN_DEFINITION, csvExportSettings } from './blotterUtils'
+import { Context } from 'openfin-fdc3'
+
+const broadcastCurrencyPairs = (symbol: string) => {
+  if (typeof fin === 'undefined') {
+    return
+  }
+  const currencyPairContext: Context = {
+    type: 'fdc.instrument',
+    name: symbol,
+    market: 'currency',
+    id: {
+      id: symbol,
+    },
+  }
+  const fdc3 = require('openfin-fdc3')
```

Comment:
Here I use require because `openfin-fdc3` is not compatible with web. Importing fdc3 results in errors where the site says it doesn't know what `fin` is. Imports are hoisted and at the time of process led to the error.